cg/constants/delivery.py

```diff
@@ -168,6 +169,9 @@
     {AlignmentFileTag.CRAM_INDEX},
 ]
 
+TAXPROFILER_ANALYSIS_CASE_TAGS: list[set[str]] = [{HermesFileTag.CLINICAL_DELIVERY}]
+
+TAXPROFILER_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [{HermesFileTag.CLINICAL_DELIVERY}]
```

Comment:
why are these two constants having the same content?

Reply:
Because our current delivery API needs of this logic. Hopefully we'll remove it soon 😅